android/package.json

```diff
@@ -16,7 +16,8 @@
     "capacitor/build.gradle",
     "capacitor/lint-baseline.xml",
     "capacitor/lint.xml",
-    "capacitor/src/main/"
+    "capacitor/src/main/",
+    "capacitor/src/main/assets/native-bridge.js"
```

Comment:
Since it's committed, no need to add the line as everything inside main is already published